lib/widgets/movie/movie_presentation.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/shared/poster.dart';
+import 'package:movie_card/widgets/movie/movie_details.dart';
+
+class MoviePresentation extends StatelessWidget {
+  const MoviePresentation({
+    super.key,
+    required this.posterName,
+    required this.title,
+    required this.date,
+    required this.percent,
+    required this.backdropName,
+    required this.genres,
```

Comment:
This is a movie right? Instead to add every parameter you can use directly the movie instance